### PR TITLE
Fix phone number

### LINKED ISSUE #23901: Sketchy phone number in `a` example
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a

### What specific section or headline is this issue about?

The US phone number 

### What information was incorrect, unhelpful, or incomplete?

> `<a href="tel:+1(555)5309">(555) 5309</a>`

### What did you expect to see?

`<a href="tel:+1(555)555-0123">(555) 555-0123</a>`

### Do you have any supporting links, references, or citations?

> Only 555-0100 through 555-0199 are now specifically reserved for fictional use; the other numbers have been reserved for actual assignment.

https://en.m.wikipedia.org/wiki/555_(telephone_number)#cite_note-2

### Do you have anything more you want to share?

The `nowhere@mozilla.org` is an interesting promise (there's a real postmaster there). I'd use `@example.org`.

### PR DESCRIPTION
**Pull request by teoli2003**
Fixes #23901

### CODE REVIEW

**Comment by bsmth** [comment]
Taking a look 👀

File: files/en-us/web/html/element/a/index.md [hunk 1]
```diff
@@ -179,7 +179,7 @@ For details about `mailto:` URLs, such as including a subject or body, see [Emai
 
 ```html
 <a href="tel:+49.157.0156">+49 157 0156</a>
-<a href="tel:+1(555)5309">(555) 5309</a>
+<a href="tel:+1(555)555-0123">(555) 555-0123</a>
```
Comment (bsmth) [inline files/en-us/web/html/element/a/index.md:182]: Should we change to 800? Based on https://developers.google.com/style/examples#phone-numbers

```suggestion
<a href="tel:+1(800)555-0123">(800) 555-0123</a>
```

**Comment by teoli2003** [comment]
I don't think it matters as 555-01xx numbers are reserved for all area codes, but having 800 will prevent people from being eventually charged if they try it.

**Comment by bsmth** [comment]
> I don't think it matters as 555-01xx numbers are reserved for all area codes, but having 800 will prevent people from being eventually charged if they try it.

OK good to know, tnx.

**Review by bsmth — APPROVED** [review]
Looks good 👍🏻

**Comment by jsoref** [comment]
Fwiw, all area codes in [NANP](https://en.m.wikipedia.org/wiki/North_American_Numbering_Plan) currently reserve +1-xxx-555-01yy -- https://en.m.wikipedia.org/wiki/555_(telephone_number), but there's nothing wrong w/ using 800 for xxx, and given how strange the 555 range is for other things, picking 800 seems like a fairly good choice.

**Pull request merged**